ui.tests/test-module/specs/wizard/wizard.authoring.spec.js

```diff
@@ -68,85 +69,52 @@ describe('Page - Authoring', function () {
-            cy.get(wizardBlockBemSelector+'__editdialog').contains('Help Content').click({force:true});
-            cy.get(wizardBlockBemSelector+'__editdialog').contains('Accessibility').click({force:true});
-            cy.get('.cq-dialog-cancel').click({force:true});
-            cy.deleteComponentByPath(wizardLayoutDrop) ;
-        });
-
         it('verify Basic tab in edit dialog of Wizard',function (){
             dropWizardInContainer();
-            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + wizardEditPathSelector);
-            cy.invokeEditableAction(editDialogConfigurationSelector);
-            cy.get(wizardBlockBemSelector+'__editdialog').contains('Help Content').click({force:true});
-            cy.get(wizardBlockBemSelector+'__editdialog').contains('Basic').click({force:true});
-            cy.get("[name='./name']").should("exist");
-            cy.get("[name='./jcr:title']").should("exist");
-            cy.get("[name='./layout']").should("exist");
-            cy.get("[name='./dataRef']").should("exist");
-            cy.get("[name='./visible']").should("exist");
-            cy.get("[name='./enabled']").should("exist");
-            cy.get('.cq-dialog-cancel').click({force:true});
-            cy.deleteComponentByPath(wizardLayoutDrop) ;
-        });
-
-
-
-        it('verify adding componenets in wizard',function(){
-            dropWizardInContainer();
-            addComponentInWizard("Adaptive Form Number Input", afConstants.components.forms.resourceType.formnumberinput);
-            addComponentInWizard("Adaptive Form Text Box", afConstants.components.forms.resourceType.formtextinput);
-            cy.get(numberInputDataPath).should("exist");
-            cy.get(textInputDataPath).should("exist");
+            cy.openEditableToolbar(sitesSelectors.overlays.overlay.component + wizardEditPathSelector).then(()=>{
+                cy.invokeEditableAction(editDialogConfigurationSelector).then(()=>{
+                    cy.get(wizardBlockBemSelector+'__editdialog').contains('Help Content').click({force:true});
+                    cy.get(wizardBlockBemSelector+'__editdialog').contains('Basic').click({force:true});
+                    cy.get("[name='./name']").should("exist");
+                    cy.get("[name='./jcr:title']").should("exist");
+                    cy.get("[name='./layout']").should("exist");
+                    cy.get("[name='./dataRef']").should("exist");
+                    cy.get("[name='./visible']").should("exist");
+                    cy.get("[name='./enabled']").should("exist");
+                    cy.get('.cq-dialog-cancel').click({force:true});
+                });
+            });
             cy.deleteComponentByPath(wizardLayoutDrop) ;
         });
 
         // todo: flaky
-        it('verify Navigation Working between tabs in Authoring', { retries: 3 }, function(){
+        it.only('verify Navigation Working between tabs in Authoring', { retries: 3 }, function(){
```

Comment:
pls remove "only" otherwise other TCs may not execute that will be added in future